website/docs/en/config/output.mdx

```diff
@@ -9,6 +9,38 @@ Used to indicate how and where Rspack outputs the contents of the generated file
 
 - **Type:** `Object`
 
+## output.amdContainer
+
+- **Type:** `string`
+
+:::warning
+Please use [`output.library.amdContainer`](#outputlibraryamdcontainer) instead as we might drop support for `output.amdContainer` in the future.
+:::
+
+Use a container(defined in global space) for calling `define`/`require` functions in an AMD module.
+
+:::warning
```

Comment:
Can we merge to two warning blocks? 

Too many warning blocks is not good for reading, we can merge them into a list:

```md
:::warning

- Please use [`output.library.amdContainer`](#outputlibraryamdcontainer) instead as we might drop support for `output.amdContainer` in the future.
- Note that the value of `amdContainer` **must be** set as a global variable.

:::
```